src/components/ParameterTypeRegistrationModal/index.tsx

```diff
@@ -13,38 +39,92 @@ const ParameterRegistrationModal = forwardRef<ParameterTypeRegistrationModalRef,
 		const [factorParameter, setFactorParameter] = useState<number>(0);
 		const [typeParameter, setTypeParameter] = useState<string>("");
 		const [idStation, setIdStation] = useState<string>("");
-		const [isLoading, setIsloading] = useState<boolean>(false);
+		const [newParams, setNewParams] = useState<ParamInterface[]>([]);
+		const [newParamsElement, setNewParamsElement] = useState<JSX.Element[]>([]);
+		const [oldParams, setOldParams] = useState<ParamInterface[]>([]);
+		const [allParams, setAllParams] = useState<ParamInterface[]>([]);
+
+		useEffect(() => {
+			getParameters();
+		}, []);
 
 		const closeModal = () => {
+			setNewParams([]);
+			setOldParams([]);
+			setNewParamsElement([]);
 			setIsDisabled(true);
 		};
 
+		const getParameters = async () => {
+			const response = await ParamRequests.getParams();
+			if (response === "error") {
+				alert("Não foi possível buscar por parâmetros cadastrados");
+			} else {
+				setAllParams(response);
+			}
+		};
+
 		const createParameter = async () => {
-			setIsloading(true);
-			const payload = {
+			const params: ParamInterface[] = oldParams.concat(newParams);
+			const response = await ParamRequests.createParam(params);
+			if (response !== "error") {
+				closeModal();
+				alert("Parâmetro cadastrado com sucesso!");
+				window.location.reload();
```

Comment:
Precisa dar reload na página para fechar o modal ? não tem alguma função para fazer isso ?

Reply:
Eu to dando reload p atualizar os gráficos que precisam ser exibidos, tem algum problema?

Reply:
Da pra fazermos de outra forma, se quiser podemos ver depois. Vou aprovar